tests/unit/test_client.py

```diff
@@ -223,14 +234,6 @@ def validate_headers(request: httpx.Request):
             )
             c.query(fql("just a mock"))
 
-        with subtests.test("should allow custom header"):
-            c = Client(http_client=http_client)
-            expected = {"yellow": "submarine"}
-            c.query(
-                fql("just a mock"),
-                QueryOptions(additional_headers=expected),
-            )
-
```

Comment:
Why remove this test?

Reply:
This test is about client configuration, not query, and they're also covered in a test about query options.

Reply:
Although I did just notice that additional headers specifically isn't covered in the query options test. The rest are, though.